Source/WebCore/loader/cache/CachedResource.h

```diff
@@ -24,6 +24,7 @@
 
 #include "CacheValidation.h"
 #include "CachedResourceClient.h"
+#include "ContentSecurityPolicy.h"
```

Comment:
Would be better to factor this so we don't have to add the include to this header, if we can. I think it should be added to the .cpp file, not the header.

Reply:
Removed it entirely, as this was a spurious leftover